packages/wifi_iot/android/src/main/java/com/alternadom/wifiiot/WifiIotPlugin.java

```diff
@@ -1510,24 +1578,38 @@ private Boolean connectToDeprecated(
     if (!enabled) return false;
 
     boolean connected = false;
+    int networkId = -1;
```

Comment:
This change looks unnecessary, it is not being used outside the for loop scope.

Reply:
Yeah the declaration can be done inside the for loop instead.